src/main/java/http/routes/EchoBody.java

```diff
@@ -14,7 +11,7 @@ public class EchoBody implements Route {
 
     public EchoBody() {
         CRLF = "\r\n";
```

Comment:
Since you are no longer using this (as its being handled in responseBuilder), it can be removed (here and in other routes classes).

Reply:
Removed from classes where it is not being used.